nmstatectl/nmstatectl.py

```diff
@@ -354,6 +365,36 @@ def run_varlink_server(args):
         logging.exception(exception)
 
 
+def run_gen_config(args):
+    for state_file in args.file:
```

Comment:
i think that this loop is obsolete, you are also printing the output starting from line 375

Reply:
It was leftover. Thanks. Fixed.